tools/arriba.wdl

```diff
@@ -0,0 +1,315 @@
+                "yes",
+                "no",
+                "reverse"
+            ]
+        }
+        mark_duplicates: "Mark duplicates in the input BAM file with Arriba. Default is true."
+        report_additional_columns_discard: "Report additional columns ['fusion_transcript', 'peptide_sequence', 'read_identifiers'] in the discarded fusions file. Default is false."
+        fill_gaps: "Fill gaps in assembled transcripts with reference bases. Expands the fusion sequence to the complete sequence of the fusion gene. Default is false."
+        max_e_value: "Maximum E-value for read support. Default is 0.3."
+        max_mismappers: "Maximum fraction of mismapped reads in the fusion region. Default is 0.8."
+        max_homolog_identity: "Maximum fraction of homologous sequence for genes. Default is 0.3."
+        max_kmer_content: "Maximum fraction of repetitive 3-mer content in the fusion region. Default is 0.6."
+        max_mismatch_pvalue: "Maximum p-value for mismatches in the fusion region. Default is 0.01."
+        quantile: "Genes with expression above the given quantile are eligible for filtering. Default is 0.998."
+        exonic_fraction: "Minimum fraction of exonic sequence between breakpoints. Default is 0.33."
+        coverage_fraction: "Minimum fraction of viral contig transcription. Default is 0.05."
+        min_itd_allele_fraction: "Minimum supporting read fraction for internal tandem duplications. Default is 0.07."
+        max_genomic_breakpoint_distance: "With 'wgs_svs', threshold for relating genomic and transcriptomic events. Default is 1000000."
+        min_supporting_reads: "Minimum number of supporting reads for a fusion. Default is 2."
+        homopolymer_length: "Maximum homopolymer length adjacent to breakpoints. Default is 6."
+        read_through_distance: "Minimum distance between breakpoints for read-through events. Default is 10000."
+        min_anchor_length: "Minimum anchor length for split reads. Default is 23."
+        many_spliced_events: "Recover fusions with at least this many spliced breakpoints. Default is 4."
+        fragment_length: "For single-end data, this is the fragment length. With paired-end reads, this is ignored and determined automatically. Default is 200."
+        max_reads: "Subsample fusions with more than this number of reads. Default is 300."
+        top_n: "Only report the top N most highly expressed viral integration sites. Default is 5."
+        max_itd_length: "Maximum length of internal tandem duplications. Default is 100."
+        min_itd_supporting_reads: "Minimum number of supporting reads for internal tandem duplications. Default is 10."
+        modify_memory_gb: "Add to or subtract from dynamic memory allocation. Default memory is determined by the size of the inputs. Specified in GB."
+        modify_disk_size_gb: "Add to or subtract from dynamic disk space allocation. Default disk size is determined by the size of the inputs. Specified in GB."
+    }
```

Comment:
This applies to a bunch of above parameters:
It's best to not refer to the default of a setting in the parameter_meta (unless there's a special circumstance*). This is because `wdldoc`/`sprocket doc` (which will hopefully be live sometime soon) pull the default value out of the WDL code and put it right next to the meta information.
So there are 2 problems with writing the default in the meta: 1) it could be wrong or out-of-date 2) it's going to be specified right there anyway so its redundant.

*a "special circumstance" for when to redundantly write the default is mostly a gut judgement. I can think of 2 cases where it makes sense. One would be where the type of the parameter is "weird" and talking about the default is a bit of translation from code to English (like the FlagFilter objects in #133 ). Two would be if _our_ default is different from _the tool's_ default.
For two, we can have a conversation if we want to keep this habit up, but I think it can prevent headaches for us, the developers to have that documented. Maybe it should move to a comment in the code rather than the meta? IDK. But I'd argue "good practice to note where we deviate from a tool's default behavior".